[debug] > Exec(__runWatch console0, Some(bbefd02c-206a-47e0-8d58-fd44b1dbef09), Some(CommandSource(console0)))
[debug] > Exec(sbtStashOnFailure, None, Some(CommandSource(console0)))
[debug] > Exec(__preWatch console0, None, Some(CommandSource(console0)))
[debug] > Exec(run, None, Some(CommandSource(console0)))
[debug] Evaluating tasks: Compile / run
[debug] Running task... Cancel: Signal, check cycles: false, forcegc: true
[info] compiling 1 Scala source to C:\Users\jaber.abbas\workspace\scala\learn\target\scala-3.4.2\classes ...
[warn] -- [E190] Potential Issue Warning: C:\Users\jaber.abbas\workspace\scala\learn\src\main\scala\main.scala:106:22 
[warn] 106 |  val p1: Unit = Point(10, 10)
[warn]     |                 ^^^^^^^^^^^^^
[warn]     |Discarded non-Unit value of type com.learn.Point. You may want to use `()`.
[warn]     |
[warn]     | longer explanation available when compiling with `-explain`
[warn] one warning found
[info] done compiling
[success] Total time: 1 s, completed 11 Jul 2024, 15:53:17
[debug] > Exec(resumeFromFailure, None, Some(CommandSource(console0)))
[debug] > Exec(sbtPopOnFailure, None, Some(CommandSource(console0)))
[debug] > Exec(__postWatch console0, None, Some(CommandSource(console0)))
[debug] > Exec(resumeFromFailure, None, None)
[debug] > Exec(__waitWatch console0, None, None)
